MAINTAINERS.md

```diff
@@ -1,3 +1,19 @@
+<img src="https://opensearch.org/assets/brand/SVG/Logo/opensearch_logo_default.svg" height="64px"/>
+
+- [Maintainers](#maintainers)
+- [Zenhub Process Workflow](#zenhub-process-workflow)
+  - [What is Zenhub](#what-is-zenhubhttpswwwzenhubcom)
+  - [Dev Deployment](#dev-deployment)
+  - [Setting up Zenhub](#setting-up-zenhub)
+  - [Sprint Board](#sprint-board-aka-zenhub-board-aka-kanban-board)
```

Comment:
I think this needs to be updated `#sprint-board-aka-zenhub-board-aka-kanban-board` doesn't seem like it will resolve